tools/buildtools/ambertools/parmchk2.xml

```diff
@@ -0,0 +1,92 @@
+<tool id="parmchk2" name="ParmChk2" version="@VERSION@">
+  <description>Amber's parameter checker</description>
+  <macros>
+    <import>macros.xml</import>
+  </macros>
+  <expand macro="requirements"></expand>
+  <command detect_errors="exit_code">
+    <![CDATA[
+        parmchk2 -i '$input1' -f '$input_format.selected_format' -s '$allparams.s' -a '$allparams.a' -o '$output1'
+        ]]>
+  </command>
+  <inputs>
+    <conditional name="input_format">
+      <param name="selected_format" type="select" label="Select a file format:">
+        <option value="mol2">mol2</option>
+      </param>
+      <when value="mol2">
+        <param type="data" name="input1" format="mol2"/>
+      </when>
+    </conditional>
+    <param name="selected_output_format" type="select" label="Select an output file format:" help="text recommended for now and it will contain frcmod data (-o)">
```

Comment:
Do you have a long-term solution in mind here? As it stands currently we could just remove this option.

Reply:
Agreed I will do as suggested for antechamber - <param type="data" name="input1" format="pdb,mol2"/>